packages/phone-number-privacy/signer/src/domain/endpoints/sign/action.ts

```diff
@@ -125,7 +131,7 @@ export class DomainSignAction implements Action<DomainRestrictedSignatureRequest
         this.io.sendFailure(res.error, res.status, session.response)
       }
     } catch (error) {
-      session.logger.error('Failed to get signature for a domain', error)
+      session.logger.error(error, 'Failed to get signature for a domain')
```

Comment:
should this be 'err'?

Reply:
when you directly pass in an `Error` type object as the first param to the logger it parses it and automatically reads the error into `err`